libecs/include/pivot/ecs/Core/Systems/description.hxx

```diff
@@ -21,6 +21,8 @@ struct Description {
 
     /// system name
     std::string name;
+    /// entity name
+    std::string entityName;
```

Comment:
Which entities name is it ? Why did you add only one name, even though the system can work on multiple entities ?

Reply:
Actually, it doesn't.
IIRC, a system may only take a single entity as parameter for now, as suggests this bit of code, in ecs::systems::Description.
`    std::vector<std::string> systemComponents;
`
Which is a vector representing the components THE (only) entity parameter has.
@Maxteke

Reply:
Yes, the system runs on only one entity, but the event itself can provide some entities
Anyway, why include the entity name, which is just the name used to interact with it *inside* the system ? When *using* the system, the name of the entity is not useful.